### PR TITLE
Allow feature files in .csv/.tsv/.xlsx format

### PR DESCRIPTION
**Pull request by aloukina**
- Feature files can now be supplied in .csv/.tsv./.xlsx format
- Added deprecation warning
- Added warning for an undocumented case when the user specified both feature subset file and feature file
- Updated the tests
- Updated documentation

### CODE REVIEW

File: tests/json2csv.py [hunk 1]
```diff
@@ -0,0 +1,99 @@
...
+                    json.dump(config_dict, outfile, indent=4, separators=(',', ': '))
+
+
+
+
+
+
+if __name__ == '__main__':
+    parser = argparse.ArgumentParser(prog='json2csv.py')
+    parser.add_argument('--all',
```
Comment (desilinguist) [inline tests/json2csv.py]: Yeah, looks like we can have options that are suppressed from the help message (`help=argparse.SUPPRESS`). So, we should probably use that functionality here.

**Comment by desilinguist** [comment]
I forgot to add tests for the feature json conversion script which probably led to the lower coverage. I will add that tomorrow morning.

**Comment by desilinguist** [comment]
@aloukina this is now ready for your review.

File: doc/column_selection_rsmtool.rst [hunk 2]
```diff
@@ -69,33 +57,37 @@ A transformation that should be applied to the column values before using it in
...
 For more advanced users, ``rsmtool`` offers the ability to assign columns to named subsets in a data file in one of the :ref:`supported formats <input_file_format>` and then select a set of columns by simply specifying the name of that pre-defined subset.
 
-If you want to run multiple ``rsmtool`` experiments, each choosing from a large number of features, generating a separate :ref:`json file <json_column_selection>` for each experiment listing columns to use can quickly become tedious.
+If you want to run multiple ``rsmtool`` experiments, each choosing from a large number of features, generating a separate :ref:`feature file <feature_list_column_selection>` for each experiment listing columns to use can quickly become tedious.
 
 Instead you can define feature subsets by providing a subset definition file in one of the :ref:`supported formats <input_file_format>` which lists *all* feature names under a column named ``Feature``. Each subset is an additional column with a value of either ``0`` (denoting that the feature does *not* belong to the subset named by that column) or ``1`` (denoting that the feature does belong to the subset named by that column).
```
Comment (aloukina) [inline doc/column_selection_rsmtool.rst]: 'Feature' -> 'feature' in line with the example below

**Pull request merged**